website/static/js/tests/fangorn.test.js

```diff
@@ -376,23 +369,23 @@ describe('fangorn', () => {
         });
 
         describe('showDeleteMultiple', () => {
-            it('no edit permissions returns false', () => {
+            it('do not show multi delete if no edit permissions', () => {
                 folder = getItem('folder', 2);
                 folder.data.permissions.edit = false;
                 item = getItem('file', 3);
                 item.data.permissions.edit = false;
                 assert.equal(Fangorn.showDeleteMultiple([folder, item]), false);
             });
 
-            it('one edit permissions returns true', () => {
+            it('do show multi delete if edit permissions for at least one selected', () => {
```

Comment:
'It does show multi delete' rather than 'it do show multi delete'. Also the similar ones.